README.md

```diff
@@ -34,82 +34,37 @@ To deploy:
 npm run migrate
 ```
 
-The script will look in for a clone of the TypeScript repo in "../TypeScript", or "./TypeScript" to move the generated files in. 
+The script will look in for a clone of the TypeScript repo in "../TypeScript", or "./TypeScript" to move the generated files in.
 
 ## Contribution Guidelines
 
-The `dom.generated.d.ts`, `webworker.generated.d.ts` and `dom.iterable.generated.d.ts` files from the TypeScript repo are used as baselines.
+The files in the `baselines/` directory from the TypeScript repo are used as baselines.
 For each pull request, we will run the script and compare the generated files with the baseline files.
 In order to make the tests pass, please update the baseline as well in any pull requests.
 
-It's recommended to first check which spec the wrong type belongs to. Say we are to update `IntersectionObserver` which belongs to [`Intersection Observer`](https://www.w3.org/TR/intersection-observer/) spec, and then we can do:
+### When the type is missing
 
-1. First check we have the spec name `Intersection Observer` in `inputfiles/idlSources.json`. If not, add it.
-2. Run `npm run fetch-idl "Intersection Observer" && npm run build && npm run baseline-accept`.
+It's possible that the automated algorithm decided that it's not well supported by browsers and thus removed it. Say we we want to add a new interface named `Foo`. Check if there is a document about that interface in [MDN](https://developer.mozilla.org/). If there is, see the browser compatibility section and check whether it's supported by two or more browsers. (Chrome, Firefox, and Safari are the relevant browsers. Supports from multiple Chromium-based browsers are counted as one support.)
```

Comment:
```suggestion
It's possible that the automated algorithm decided that it's not well supported by browsers and thus removed it. Say we we want to add a new interface named `Foo`. Check if there is a document about that interface in [MDN](https://developer.mozilla.org/). If there is, see the browser compatibility section and check whether it's supported by two or more browsers. (Chrome, Firefox, and Safari are the relevant browsers. Support from multiple Chromium-based browsers are counted as one support.)
```

Reply:
Maybe "Support from multiple Chromium-based browsers are counted as one browser" - using support at the end  doesn't really fit in English grammar.